setup.py

```diff
@@ -1,66 +0,0 @@
-#!/usr/bin/python
-# coding: utf-8
-
-"""
-Allow the project to be installed as a local package, using :
-    python -m pip install .
-    or
-    python -m pip install -e .
-The -e option stands for editable, which allows to change the
-source code of the package without having to reinstall it. Using this option
-will add the current directory (.) to the system path.
-"""
-
-from setuptools import setup, find_packages
-
-requires = [
-    'importlib_resources>=5.10.2,<5.11',
-    'tomli>=2.0.1,<2.1',
-    'networkx>=2.0,<3.0', # a workaround to nipype's bug (issue 3530)
```

Comment:
this requirement was removed and switched to bumping up the nipype requirement instead as this is now fixed